tools/topology2/machine.conf

```diff
@@ -0,0 +1,78 @@
+
+#
+# Simple Machine - High level topology - Maps to machine driver.
+#
+#
+# PCM host -> buffer0 -> volume -> buffer1 - > SSP0
+#
+
+include("volume.conf")
+include("buffer.conf")
+include("ssp.conf")
+include("icl.conf")
+include("pipeline-volume.conf")
+
+#
+# Pipeline definitions
+#
+
+pipelines [
+
+	# Use volume pipeline and connect to SSP0.
+	"pipeline-volume.0" {
+		format	[
+			"s16le"
+			"s24le"
+			"s32le"
+		]
+
+		rates [
+			"48000"
+		]
+
+		channels [
+			"2"
+		]
+
+		schedule_period_us	"1000"
+		schedule_time_domain	"timer"
+
+		# connects sink to SSP.0
+		endpoint."sink.0" {
+			dai "ssp.0"
```

Comment:
is this adding a `dai "ssp.0"` entry to the endpoint definition from pipeline.conf, thus making it effectively
```
	DefineEndpoint."sink.$INDEX.0" {
		name		"buffer.$INDEX.1"
		complete	"false"
		dai		"ssp.0"
	}
```
?